src/services/doppler-api-client.double.ts

```diff
@@ -69,6 +69,13 @@ const campaignDeliveryItems = [...Array(100)].map((_, index) => {
     campaignSubject: '¿Como sacarle provecho a la primavera?',
     deliveryStatus: getDeliveryStatus(Math.round(Math.random() * (5 - 1) + 1)),
     clicksCount: Math.round(Math.random() * (100 - 1) + 1),
+    links: [
+      {
+        href: 'http://dopplerfilesint.fromdoppler.net/Users/50018/Campaigns/33850437/33850437.png',
```

Comment:
Here you can have several links that have nothing to do whith preview, perhaps add one like that, and a filter in map funcion. What do you think?

Reply:
maybe we can use in the future this links. for now we can keep it